src/main/java/com/epam/preprod/sirenko/UniqueObjectsList.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.preprod.sirenko;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class UniqueObjectsList extends ArrayList<Object> {
+	private static final String MESSAGE_CANT_ADD_ELEMENT = "Can't add such element";
+	private static final String MESSAGE_CANT_ADD_ELEMENTS = "Can't add such elements";
+	
+	/**
+	 * Appends the specified element to the end of the container
+	 **/
+	@Override
+	public boolean add(Object object) {
+		if (super.contains(object)) {
```

Comment:
We can use contains without super here and in other methods

Reply:
Fixed